pymathics/graph/base.py

```diff
@@ -725,14 +722,16 @@ def _graph_from_list(rules, options, new_vertices=None):
             new_edges, new_edge_properties, options=options, new_vertices=new_vertices
         )
 
-
+    
```

Comment:
This is the only function that I actually was working on. Other changes come from #9.